### PR TITLE
DBZ-3936 Upgrade MySQL example image to 8.0

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-3936

### CODE REVIEW

File: examples/mysql/1.7/mysql.cnf [hunk 1]
```diff
@@ -43,4 +43,7 @@ log_bin           = mysql-bin
 expire_logs_days  = 1
 binlog_format     = row
 
+# Configures the amount of table metadata added to the binary log when using row-based logging
```
Comment (gunnarmorling) [inline examples/mysql/1.7/mysql.cnf:46]: I don't think we should change this here; instead, we could provide a specific mysql.conf file just for the single integration test (using Testcontainers) related to invisible columns.

**Comment by gunnarmorling** [comment]
@jpechane, if we change this image to 8.0, do we need any changes in CI/testing set-up? Not that we end up testing twice with 8.0 but not at all with 5.7.

**Comment by jpechane** [comment]
There is no relation to CI - MySQL example images are not used for testing.

**Comment by gunnarmorling** [comment]
Ok, thanks for clarifying, @jpechane. Applied. Thanks, @ani-sha!

**Pull request merged**